src/plugins/visualizations/public/embeddable/create_vis_embeddable_from_object.ts

```diff
@@ -88,6 +89,8 @@ export const createVisEmbeddableFromObject = (deps: VisualizeEmbeddableFactoryDe
 
     const editable = getCapabilities().visualize.save as boolean;
 
+    const savedAugmentVisLoader = getSavedAugmentVisLoader();
```

Comment:
If we are calling on getSavedAugmentVisLoader() inside the visualizations plugin don't we need to first set it in here: `src/plugins/visualizations/public/plugin.ts` @joshuarrrr

Reply:
Thanks, yes this was a miss. You've opened https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3616 so I'll resolve here.